src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,87 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File inputFile = new File(fromFileName);
+        File uptputFile = new File(toFileName);
+        writeToFile(uptputFile, createReport(readCsvFile(inputFile)));
+    }
+
+    public void writeToFile(File file, String data) {
+        FileWriter fileWriter;
+        try {
+            fileWriter = new FileWriter(file);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        try (BufferedWriter bufferedWriter = new BufferedWriter(fileWriter)) {
+            bufferedWriter.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Buffered writer cant access Writer", e);
+        }
+
+    }
 
+    public String createReport(String[][] data) {
+        StringBuilder report = new StringBuilder();
+        int i = 0;
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        while (data[i][0] != null) {
```

Comment:
```suggestion
        String[] records = dataFromFile.split(some pattern);
        for (String record: records)
```

Reply:
I do not understant it, do not we have to make sure not ho hit null in array of data from file?

Reply:
two dimentional array Changed to StringBuilder